app/helpers/application_widgets_helper.rb

```diff
@@ -90,6 +90,9 @@ def delete_button(name, options = nil, html_options = nil, &block)
   # Display a progress_bar with the given percentage and styling. The percentage is assumed to
   # be a number ranging from 0-100. In addition, a block can be passed to add custom text.
   #
+  # Note: I18n translations used in the block should state the full localization path. This
+  # is because a layout is rendered for this method.
```

Comment:
one way around this is to instance_exec within the block. do you want to have a go in a separate PR?